### DISCUSSION TITLE
Having the Black-  & Whitelists centrally managed (on Github)

### DISCUSSION THREAD

**Discussion by christophkutz** [Romanitho/Winget-AutoUpdate#345] [Ideas]
Hi,

we are currently evaluating this tool for our company and already successfully deployed this in on some test systems via Intune.
To reduce future maintenance in regards of Black &  Whitelists it would be great if there is an option to have this centrally managed, for example in an own Gihub repo.

An Example in Powershell could be 
`Invoke-WebRequest https://raw.githubusercontent.com/Romanitho/Winget-AutoUpdate/main/excluded_apps.txt `

**Comment by Romanitho** [comment]
Hello. This feature already exists 😊

**Reply by Romanitho** [reply]
In your context, just install with `-ListPath https://raw.githubusercontent.com/Romanitho/Winget-AutoUpdate/main/`
Change URL with yours, without file name. It will automatically try to download file regarding your config.
